notebook/static/notebook/js/notebook.js

```diff
@@ -2844,7 +2844,100 @@ define([
             this._checkpoint_after_save = false;
         }
     };
-    
+
+    Notebook.prototype.save_notebook_as = function() {
+        // If current notebook has some changes, save them, or the copied notebook won't have them.
```

Comment:
Is this still needed? We're sending a new copy of the notebook to the new name, so I don't think we need to save it at the old name again first.